crates/katana/storage/provider/tests/block.rs

```diff
@@ -145,6 +173,109 @@ where
 
         assert_eq!(actual_receipts.as_ref().map(|r| r.len()), Some(expected_block.body.len()));
         assert_eq!(actual_receipts, Some(receipts));
+        assert_eq!(actual_executions, Some(executions));
+
+        assert_eq!(actual_block_tx_count, Some(expected_block.body.len() as u64));
+        assert_eq!(actual_state_root, Some(expected_block.header.state_root));
+        assert_eq!(actual_block_txs, Some(expected_block.body.clone()));
+        assert_eq!(actual_block_hash, Some(expected_block_hash));
+        assert_eq!(actual_block, Some(expected_block));
+    }
+
+    Ok(())
+}
+
+fn insert_block_empty_test_impl<Db>(provider: BlockchainProvider<Db>, count: u64) -> Result<()>
```

Comment:
generally this looks alright but im curious, is there a specific case that we're testing this for ?

Reply:
I was just thinking that don't have much test for default values and empty stuff. And wanted to ensure we test in those cases `None` being returned correctly.
I've actually replaced the transaction hash by `FieldElement::ZERO`, to ensure default values are not causing any non desired effect in c373d4b9fb3649a19206ae2de9073991b41a059b.